share/p2p/shrexnd/server.go

```diff
@@ -39,13 +38,12 @@ type Server struct {
 }
 
 // NewServer creates new Server
-func NewServer(params *Parameters, host host.Host, store *eds.Store, getter share.Getter) (*Server, error) {
+func NewServer(params *Parameters, host host.Host, store *eds.Store) (*Server, error) {
```

Comment:
Because the server does not have access to the Header, correct?

Reply:
wdym?

Reply:
This is because we need to use the store directly now instead of implicit StoreGetter